chainreg/chainregistry.go

```diff
@@ -105,6 +105,10 @@ type Config struct {
 	// DBTimeOut specifies the timeout value to use when opening the wallet
 	// database.
 	DBTimeOut time.Duration
+
+	// RemoteWalletDB should be set to true if we'd like to store the wallet
+	// database in the remote DB.
+	RemoteWalletDB bool
```

Comment:
Hmm, I wonder if this should just be the default or not 🤔

I guess one may want to have the wallet in the local store if they want to commit less data into the `etcd` instance?

Reply:
I think we need to remain backwards compatible in a sense that the remote DB in LND is currently just a split of the channeldb residing in the same file when using bbolt. So if we want to remain backwards compatible and keep the `wallet.db` file for local clients we need this option.